docs/guides/webview-versions.md

```diff
@@ -0,0 +1,190 @@
+
+`$(SYSTEM_VERSION_PREFIX)$(MAJOR_VERSION).$(MINOR_VERSION).$(TINY_VERSION).$(MICRO_VERSION).$(NANO_VERSION)`
+
+### `SYSTEM_VERSION_PREFIX`
+
+Here is what the `SYSTEM_VERSION_PREFIX` values mean:
+
+| macOS version | SYSTEM_VERSION_PREFIX |
+| ------------- | --------------------- |
+| 10.11         | 11                    |
+| 10.12         | 12                    |
+| 10.13         | 13                    |
+| 10.14         | 14                    |
+| 10.15         | 15                    |
+| 11.0          | 16                    |
+| 12.0          | 17                    |
+| 13.0          | 18                    |
+| sdk=iphone\*  | 8                     |
+
+### Example
+
+The webkit version shipped with Safari 15.5 on macOS Monterey (12.x) has the following version number: `17613.2.7.1.8`.
+
+| part                    | version |
+| ----------------------- | ------- |
+| `SYSTEM_VERSION_PREFIX` | 17      |
+| `MAJOR_VERSION`         | 613     |
+| `MINOR_VERSION`         | 2       |
+| `TINY_VERSION`          | 7       |
+| `MICRO_VERSION`         | 1       |
+| `NANO_VERSION`          | 8       |
+
+</details>
+
+The tables below map OS versions to WebKit versions and the corresponding Safari equivalent, so you can use sites like [caniuse](https://caniuse.com) to figure out if a specific web platform feature is supported.
+
+### macOS
+
+On macOS Tauri uses the component that comes preinstalled with macOS since version 10.10 (Yosemite). It is considered a core component and is therefore updated with the regular OS updates. This means _unsuspported_ macOS versions also **do not** receive WebKit updates.
+
+To find the Webkit version used by WKWebView you can use this command:
+
+```shell
+awk '/CFBundleVersion/{getline;gsub(/<[^>]*>/,"");print}' /System/Library/Frameworks/WebKit.framework/Resources/Info.plist
+```
+
+|             | macOS                            | WebKit           | Safari Equivalent | Notes                                                            |
+| ----------- | -------------------------------- | ---------------- | ----------------- | ---------------------------------------------------------------- |
+| High Sierra | 10.13                            | [604.1.38.1.6]   | 11.0              |                                                                  |
+|             | 10.13.1                          | [604.3.5]        | 11.0.1            |                                                                  |
+|             | 10.13.2                          | [604.4.7.1.3]    | 11.0.2            | 27" iMacPro's received a special webkit version: [604.4.7.10.4]  |
+|             | 10.13.2 Supplemental Update      | [604.4.7.1.6]    | 11.0.2            | 27" iMacPro's received a special webkit version: [604.4.7.10.6]  |
```

Comment:
Grammatically these suggestions are weird to not have the plural and ownership, but that's Apple's guides 🤷🏽‍♂️

```suggestion
|             | 10.13.2 Supplemental Update      | [604.4.7.1.6]    | 11.0.2            | 27" iMac Pro received a special WebKit version: [604.4.7.10.6]  |
```